### PR TITLE
Local v cloud

### LINKED ISSUE #161: Resolve issue to do with cloud vs. local path issues
## Task Description

- Locally the code won't run due to the fact that locally 'os.path.exists' returns False because it does not have the permission to run os.stat()

## User Stories

- As a developer, I want my code to behave the same locally as it does in the cloud so that I can test it easier.

## Relevant Files [If Available] 

- 'extract.py'

### LINKED ISSUE #175: Add more cases to database
## Task Description

- We're currently aiming to have scraped and stored the first 100 pages of the website

## User Stories

- As a journalist, I want to have access to as many cases as possible, so that the analytics I perform are more accurate.

## Relevant Files [If Available] 

- 'pipeline.py'

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
This now works both locally and in the cloud. The difference now is the env variables. For example, in the local .env the STORAGE_FOLDER variable should be 'data', while for the cloud version (i.e. terraformed version) the variable should be '/tmp'.

Closes #161 
Closes #175

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Also added 100 pages worth of data to the db.

File: pipeline/extract.py [hunk 1]
```diff
@@ -131,13 +131,26 @@ def get_case_title(case_soup: BeautifulSoup) -> str:
...
-        f.write(response.content)
+    court_case["filepath"] = f"{ENV['STORAGE_FOLDER']}/{court_case['title']}.pdf"
+
+    try:
+        response = requests.get(court_case["pdf"], timeout=10)
+        with open(f"{court_case['filepath']}", "wb") as f:
+            f.write(response.content)
+        return
+    except (requests.exceptions.MissingSchema, requests.exceptions.ReadTimeout):
+        pass
```
Comment (MahinRahman8901) [inline pipeline/extract.py:145]: nice try and except, what dos the except check for however?

File: pipeline/extract.py [hunk 2]
```diff
@@ -204,7 +218,7 @@ def extract_cases(pages: int) -> pd.DataFrame:
 
     extracted_cases = []
 
-    for i in range(1, pages+1):
+    for i in range(start_page, end_page+1):
```
Comment (MahinRahman8901) [inline pipeline/extract.py:221]: much more pythonic!

File: pipeline/extract.py [hunk 3]
```diff
@@ -226,7 +240,8 @@ def extract_cases(pages: int) -> pd.DataFrame:
 
     for case_data in extracted_cases:
         download_pdfs(case_data)
-        parse_pdf(case_data)
+        if case_data:
+            parse_pdf(case_data)
```
Comment (MahinRahman8901) [inline pipeline/extract.py:244]: nice that youre checking if the data exists

File: pipeline/extract.py (hunk 1 above)
Comment (luciegamby-sigmalabs) [reply pipeline/extract.py:145]: It's a check in case the url given is invalid or the request times out

**Pull request merged**